### DISCUSSION TITLE
Quicumque Vult

### DISCUSSION THREAD

**Discussion by ajmalton** [DivinumOfficium/divinum-officium#2602] [Developers corner]
So I thought the rules for the Athanasian Creed were fairly simple: at Sunday Prime _quando fit officium de Dominica_.  And then that in 1910 it was reduced to Epiphanytide and after Pentecost; and then in 1955 to just Trinity.

But...

1/ We don't include it during Lent or Advent even for the oldest rubrics; and
2/ The complex condition (lines 887-882 of specials) depends on sub checksuffragium, which is curious at best.

What really is the Roman rule for saying the Athanasian Creed, through history?  Before 1910 was it in fact omitted during penitential seasons? 

(NB the underlying motivation is: what is the historical rule for including the Suffragium? Because _sub_ _checksuffragium_ is also needlessly complicated, and different for Missa and Horas, and used in rather odd ways through the code.)

**Comment by APMarcello3** [comment]
For Divino Afflatu rubrics, Section XXXIII, 2 states: "The Athanasian Creed is said at Prime before the Psalm (Bene fac) on all Sundays, when the Office is of the Sunday. But on the Sundays within the Octaves of Christmas, Epiphany, Ascension and Corpus Christi, and on Easter and Pentecost, when only three Psalms are said as on Feasts, there is no Athanasian Creed. it is said on Sundays within other Octaves and on Trinity Sunday. Otherwise it is never said even if a Double is celebrated on a Sunday. The Gloria Patri is said at the end of it."

**Comment by gregordick** [comment]
Note that the rubric that Father quotes above is still printed in Divino breviaries but is actually the rubric that was in force in 1900.  Divino afflatu modifies that rubric as follows in Additiones et variationes, VIII. (translation from [here](https://www.fisheaters.com/forums/showthread.php?tid=9340&pid=85618)):

> 2. The Athanasian Creed is added at Prime on the Feast of the Most Holy Trinity and on Sundays only after the Epiphany and after Pentecost, when of them the Office is said, saving the exception which is in the following number.
> 3. When on Sunday a Commemoration is made of a Double, or some Octave, the Suffrage, Preces, and Athanasian Creed are omitted. And on Ferias on which likewise is made a Commemoration of a Double or Octave, the Suffrage and dominical Preces at Prime at Compline are omitted; but not the ferial Preces, if they are to be said.

**Comment by ajmalton** [comment]
Well that helps but it's still not clear.

Looking at the 1888 Pustet pars hiemalis, on p. xxxiii, at XXII.4.Where it puts the Symbol _after_ the Psalms at Prime in the Sunday office, and _additur semper_.  I must suppose something changed between 1888 and 1900, though I cannot think what.  

I note have a suspicion the situation is even more complicated than d.o.’s code!  Certainly (having read further) it seems that the pre-Divino rule is not we actually do.

AJM

**Reply by gregordick** [reply]
@APMarcello3's quotation above has "before" where [this translation (big PDF)](https://www.divinumofficium.com/www/horas/Help/Rubrics/EnglishDORubrics.pdf) has "after", but otherwise agrees with it, so perhaps this was a transcription error.  [This 1942 Desclée breviary](https://archive.org/details/breviarium-romanum-1942/Breviarium_Romanum_1942_-_Pars_Hiemalis/mode/2up) agrees with the 1888 breviary that you cited and [puts the Creed after Retribue](https://archive.org/details/breviarium-romanum-1942/Breviarium_Romanum_1942_-_Pars_Hiemalis/page/n65/mode/2up).  (Note that "Bene fac" is the Bea equivalent of "Retribue".)

Regarding "additur semper", I think that might just be rhetorical flourish; the end of the sentence qualifies it with "ut infra in propria Rubrica [i.e. XXXIII.2] dicitur".  The corresponding rubrics in 1568 are [essentially](https://archive.org/details/breviarium-romanum-1568/page/n47/mode/2up) the [same](https://archive.org/details/breviarium-romanum-1568/page/n57/mode/2up).

I wouldn't be surprised if D.O. gets pre-Divino wrong; it's often the case in general.

**Comment by ajmalton** [comment]
Good, thanks for the references: now it's clear.
